lib/sorbet-rails/helper_rbi_formatter.rb

```diff
@@ -18,7 +18,7 @@ def generate_rbi
 
     @parlour.root.add_comments([
       'This is an autogenerated file for Rails helpers.',
-      'Please rerun bundle exec rake rails_rbi:helpers to regenerate.'
+      'Please rerun \"bundle exec rake rails_rbi:helpers\" to regenerate.'
```

Comment:
Because we use` ' ' `here, you don't need to escape the `"`. Currently your change generate this:
```
       +# Please rerun \"bundle exec rake rails_rbi:helpers\" to regenerate.
```
But I think we only want?
```
       +# Please rerun "bundle exec rake rails_rbi:helpers" to regenerate.
```